packages/webview_flutter/webview_flutter_wkwebview/test/src/web_kit_webview_widget_test.mocks.dart

```diff
@@ -1,5 +1,5 @@
 // Mocks generated by Mockito 5.0.17 from annotations
-// in webview_flutter_wkwebview/test/src/web_kit_webview_widget_test.dart.
+// in webview_flutter_wkwebview/example/ios/.symlinks/plugins/webview_flutter_wkwebview/test/src/web_kit_webview_widget_test.dart.
```

Comment:
This seems wrong; any idea how this path got messed up?

Reply:
I'm not sure, but I upgraded Flutter and mockito to the latest versions and it still outputs the symlink. I created an issue for the dart team: https://github.com/dart-lang/mockito/issues/521